utils/time.js

```diff
@@ -43,9 +43,16 @@ const getTimeInSecondAfter = ({ timestamp = Date.now(), days = 0, hours = 0, min
   return parseInt(timeInMilliseconds / 1000);
 };
 
+const getLast24HourTime = (timestamp) => {
+  const currentTime = timestamp;
+  currentTime._seconds -= 24 * 60 * 60;
+  return currentTime;
+};
+
```

Comment:
Is this needed? Can't we do `getTimeInSecondsAfter({ days: -1 })`?

Reply:
Not sure how this can be done. Can you please give some more context on it ?

Reply:
Conversation continued here -
https://github.com/Real-Dev-Squad/website-backend/pull/632#discussion_r920371238